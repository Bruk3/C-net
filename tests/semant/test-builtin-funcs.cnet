
int main(string[] args)
{
	int argc = args.alength();

	int arg1len = args[0].slength();

	file arg_count = fopen("argl", "w");
	socket send = nopen("www.google.com", "tcp", 80, "connect");

	arg_count.writeln(soi(arg1len));

	return 0;
}
