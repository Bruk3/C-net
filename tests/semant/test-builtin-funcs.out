int main(string[] args)
{
int argc = (int : alength((string[] : args)));
int arg1len = (int : slength((string : args[(int : 0)])));
file arg_count = (file : fopen((string : "argl"), (string : "w")));
socket send = (socket : nopen((string : "www.google.com"), (string : "tcp"), (int : 80), (string : "connect")));
(int : writeln((file : arg_count), (string : soi((int : arg1len)))));
return (int : 0);
}

